desktopexporter/app/utils/use-key-press.ts

```diff
@@ -33,3 +35,54 @@ export const useKeyPress = (targetKeys: string[]) => {
   );
   return keyPressed;
 };
+
+export const useKeyCombo = (modifierKeys: string[], targetKeys: string[]) => {
```

Comment:
Just a note that you can use a more restricted enum type for `modifierKeys` here:

```typescript
type KeyString = "Alt" | "Control" | "Meta" | "Shift";
``` 

https://www.typescriptlang.org/play?#code/C4TwDgpgBA0hIGVgCcCWA7A5lAvFARAIIA2w+UAPgQMID26Ktx5V+AshMAIYsEIAWqAGZkA3AFgAUFKEBXdAGNgqelFkBnCHBB0AtgCNaACl20AJsNQRk29QC5Y8JGiwBtALoAaKN2SZOtg7qKBiYHgCUUADeUlBQxJxQyBDquFCmFkJWNvDqAHS6XGBGJubakTgAfNGxcXX16gDuqMAK-FClZuU1kvV9dQpcmgQkZHa1-f3JwLLI6FAAjBK9k-WDw-h0DMhM+OMrq-XTs-MATMuHA0PQ7Jw8+5dHnCdQAMwXj+s3AsJjE4-HOZQAAsH0uZggQi4slID0ecUB8yhxE0YMmAF9aujwh8FPR1EwIHliLRMEZkuocVJMdJJAB6OlQABytCg1h2yCkGi08D0hiMriIpHwXiggoAKiLwlIpAyoABRZAcgCEXM02j5xkFekK6DMIu8EqlQA